.circleci/config.yml

```diff
@@ -597,6 +597,14 @@ jobs:
               --ignore src/olympia/versions/ \
               --ignore src/olympia/zadmin
 
+  make_tests:
```

Comment:
This should run as a separate test because it modifies the environment and .env files... it's theoretically possible to clean up that data after, but probably not worth the inherent risk.